torntools/scripts/content/profile/ttProfile.js

```diff
@@ -1254,7 +1254,7 @@ function ageToWords() {
 }
 
 function disableAllyAttack() {
-	if (settings.pages.profile.block_ally_attacks) {
+	if (settings.pages.profile.block_ally_attacks && doc.find("a.profile-button.profile-button-attack") !== null) {
```

Comment:
This is Javascript, `doc.find("a.profile-button.profile-button-attack")` would suffice.

Reply:
Just an old habit from Bash.